.circleci/config.yml

```diff
@@ -2,12 +2,16 @@ version: 2.1
 
 executors:
   docker_minio:
+    # An executor to run S3 compatibility unit tests
     working_directory: ~/project
     environment:
       GOOGLE_COMPUTE_ZONE: us-west2-c
     resource_class: 'medium+'
     docker:
-      - image: circleci/golang
+      - image: gcr.io/onec-co/datamon-builder:latest
+        auth:
+          username: _json_key
```

Comment:
`_json_key` resolves to a value in Circle CI, `_json_key` as a value is passed to docker, or ... ?

Reply:
`_json_key` is a special tag used by `docker login` to interpret the password as an `OAuth2` token when connecting to the `gcr.io` login. If you are interested in that part, take a look at this repo for more details: https://github.com/ivanilves/lstags/issues/211